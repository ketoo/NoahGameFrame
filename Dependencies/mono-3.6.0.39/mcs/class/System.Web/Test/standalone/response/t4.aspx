<% @Page Language="C#" %>
<%
	Response.Output.Write ("hello");
	Response.Flush ();
	Response.Buffer = false;
	Response.Output.Write ("world");
	Response.Buffer = true;
	Response.Output.Write ("oops");
	Response.Flush ();
%>