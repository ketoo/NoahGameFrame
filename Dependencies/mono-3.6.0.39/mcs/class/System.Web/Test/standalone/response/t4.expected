5
hello
5
world
4
oops
0

